docs/Running-behind-nginx.md

```diff
@@ -1,9 +1,43 @@
-# Netdata via nginx
+# Running Netdata Behind Nginx
```

Comment:
Did we decide to start using [Capitalization instead of Sentence Case](https://danielmiessler.com/blog/a-list-of-different-case-types/)?
If so, then we need to be consistent everywhere. We currently have only a couple of titles using Capitalization, which I see as bugs.

Reply:
For consistency, I agree with sentence case. 
I have changed made the change.